### ISSUE TITLE
RTX 3080 LHR Detected and never unlocks above 40 Mhs

### ISSUE THREAD

**Issue by MAlmarri98** [NebuTech/NBMiner#583]
My EVGA and ASUS Strix OC 3080, both LHR, they start mining unlocked at 74-75 Mhs consistently at 74.5% unlock in the latest version of Nbminer(39.7), but when I keep it on for a while the cards gets detected but it doesnt appear in the CMD, I think its a bug, the hashrate just drops to 40 and doesnt go back up again. The cards are still running on the same core,mem, and power wattage, just lower hashrate, that still needs to be fixed, good luck!

**Comment by DrNeoBandi** [comment]
I have also experienced the problem but only on NH when it changes algorithm. (maybe your miner makes a restart and the issue is the same)
When I start mining ETH, LHR unlock working as it should, bringing me 75mh/s (ASUS TUF RTX 3080). But when another algorithm becomes more profitable (like Octopus) and changes to that AND THEN it changes back to Dagger--> LHR unlock is simply never starting and bringing me 40mh/s. Additional info that console says that LHR mode is active, but seemingly not.
The bug occurs on both NH miner and on Miner Legacy Fork Fix as well but I do think that it has nothing to do with that and only the "restart" causes the issue.

**Comment by stegx** [comment]
Asus TUF RTX 3080 LHR same phenomenon here.

**Comment by MAlmarri98** [comment]
> I have also experienced the problem but only on NH when it changes algorithm. (maybe your miner makes a restart and the issue is the same) When I start mining ETH, LHR unlock working as it should, bringing me 75mh/s (ASUS TUF RTX 3080). But when another algorithm becomes more profitable (like Octopus) and changes to that AND THEN it changes back to Dagger--> LHR unlock is simply never starting and bringing me 40mh/s. Additional info that console says that LHR mode is active, but seemingly not. The bug occurs on both NH miner and on Miner Legacy Fork Fix as well but I do think that it has nothing to do with that and only the "restart" causes the issue.

I have seen it happen live, the Mh/s just drop from 74-75 to around 40 directly, it doesn't restart or anything, and it doesn't state that it got locked also, kinda weird but I am sure they will fix it soon.

**Comment by MAlmarri98** [comment]
> Asus TUF RTX 3080 LHR same phenomenon here.

yeah wierd, I have 2 full rigs on NBMiner, one mixed rig LHR and non-lhr and the other is x8 3060ti LHR that is running consistently without any issues, only occurs on my gaming PC's 3080.

**Comment by BluefireXD** [comment]
Same here too. 3080 ti, ~80-90MHs suddenly drops to 40MHs or even 20MHs. No lock or something happend. 
![MH Drop](https://user-images.githubusercontent.com/12106363/140399663-05ba16b3-1785-441a-b640-7f5ba92f79c1.png)

**Comment by Marsi4eg** [comment]
I also noticed hashrate problem on 3060 v1 and it also happens in NH when it changes algo from kawpow/octopus to ethash. I found a workaround - just increase miner restart delay in NH advanced settings. Default value is 500ms, I've set it to 5000 (5 sec) and the problem gone. Looks like cards just need some time to properly reset between algos or something

**Comment by DrNeoBandi** [comment]
> I also noticed hashrate drop problem on 3060 v1 (must be 33.5 unlocked, drops to 18) and it also happens in NH when it changes algo from kawpow/octopus to ethash. I found a workaround - just increase miner restart delay in NH advanced settings. Default value is 500ms, I've set it to 5000 (5 sec) and the problem gone. Looks like cards just need some time to properly reset between algos or something
       
      This workaround works! Solves either on NH and MLFF !! Thank you!

**Comment by BluefireXD** [comment]
I only use Dagger :/. Means there is not switching between algos.

**Comment by stegx** [comment]
> I also noticed hashrate drop problem on 3060 v1 (must be 33.5 unlocked, drops to 18) and it also happens in NH when it changes algo from kawpow/octopus to ethash. I found a workaround - just increase miner restart delay in NH advanced settings. Default value is 500ms, I've set it to 5000 (5 sec) and the problem gone. Looks like cards just need some time to properly reset between algos or something

This workaround didn't work for me, i even tried setting it higher than 5 sec. It's still locking to 40.

**Comment by MAlmarri98** [comment]
Ok, so I resorted to another miner for the past week, I went to try Nbminer again, its working flawlessly all of a sudden, 2 days now stable at 75Mh/s on EVGA 3080 LHR. Didn't change any OC settings nor miner settings.

**Comment by BluefireXD** [comment]
y for ~1 day everything seems fine too. lets see if it stays stable.
